<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>additionalInformation</class>
 <widget class="QMainWindow" name="additionalInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact Update</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="addInfo">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>Additional Information</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="recAttempt">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
              <property name="text">
               <string>Recruitment Attempt</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="volStat">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
              <property name="text">
               <string>Volunteering Status</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="addAttemptBut">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 155, 155);</string>
              </property>
              <property name="text">
               <string>Add Attempt</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateStatusBut">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 205, 255)</string>
              </property>
              <property name="text">
               <string>Update Status</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="closeAddInfoBut">
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="addRecAttempt">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="ftofContactLab">
              <property name="text">
               <string>Friend To Friend Contacts</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="ftofContacts"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="date">
                <property name="text">
                 <string>Date (mm/dd/yyyy)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="month">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>12</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="day">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>31</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="year">
                <property name="minimum">
                 <number>2020</number>
                </property>
                <property name="maximum">
                 <number>2050</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Was This a Success? (Did the contact volunteer after attempt?)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="yesBox">
                <property name="text">
                 <string>Yes</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="noBox">
                <property name="text">
                 <string>No</string>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="notSureBox">
                <property name="text">
                 <string>Not Sure</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Length of Attempt (How long did you communicate with the contact?)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="hrSpin"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>hr</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="minSpin"/>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>min</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Form of Contact</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="FOC_Combo">
                <item>
                 <property name="text">
                  <string>Phone</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>In Person</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Email</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Social Media</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Other</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Additional Inforamtion</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="addInfoLine"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Response (if any)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTextEdit" name="resText">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="addAttemptBut_2">
          <property name="text">
           <string>Add Recruitment Attempt</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="return1">
          <property name="text">
           <string>Return to Main</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="volStatus">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="volStatLab">
            <property name="text">
             <string>Volunteering Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="allContacts">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>400</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="stat">
              <property name="text">
               <string>Status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="activeBut">
              <property name="text">
               <string>Active</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="notActiveBut">
              <property name="text">
               <string>Not Active</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="return2">
            <property name="text">
             <string>Return to Main</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
